crates/democracy/src/types.rs

```diff
@@ -17,10 +17,10 @@ pub struct Vote<Balance> {
 }
 
 /// The account is voting directly.
-#[derive(Encode, Decode, Clone, Eq, PartialEq, Default, RuntimeDebug, TypeInfo)]
-pub struct Voting<Balance> {
+#[derive(Encode, Decode, Clone, Eq, PartialEq, Default, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub struct Voting<Votes> {
     /// The current votes of the account.
-    pub(crate) votes: Vec<(ReferendumIndex, Vote<Balance>)>,
+    pub(crate) votes: Votes,
```

Comment:
Why are you moving `Vec` out of here? I get that it's functionally identical, but why not just replace `Vec` by `BoundedVec` to minimize changed compared to upstream?